ethpm/tools/builder.py

```diff
@@ -4,18 +4,21 @@
 from typing import Any, Callable, Dict, Iterable, List, Optional, Set, Tuple
 
 import cytoolz
```

Comment:
Looks like you could go ahead and clean up the rest of these to use the `eth_utils.toolz` aliases.